apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/comparableAssertions.kt

```diff
@@ -43,3 +43,14 @@ fun <T : Comparable<T>> Expect<T>.isGreaterThan(expected: T) =
 fun <T : Comparable<T>> Expect<T>.isGreaterThanOrEqual(expected: T) =
     addAssertion(ExpectImpl.comparable.isGreaterOrEquals(this, expected))
 
+/**
+ * Expects that the subject of the assertion is equal to [expected]
+ * The comparison is carried out with [Comparable.compareTo].
```

Comment:
I think it is important that the compareTo is mentioned in the first sentence.
```suggestion
 * Expects that the subject of the assertion is equal to [expected]
 * where the comparison is carried out with [Comparable.compareTo].
```